tests/test_variables.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+
+def test_additional_variable():
+    from alfasim_sdk.variables import Variable, Visibility, Location, Scope
+
+    with pytest.raises(ValueError, match='The field "name" cannot be empty'):
```

Comment:
You can turn all the `pytest.raises` check in a separate parametrized test, where you parametrize the attribute you want to change and the expected message.

```python
@pytest.mark.parametrize('attr, 'expected_message', [
    (dict(unit='foo'), "foo is not a valid unit"),
    (dict(caption='  '), "The field "caption" cannot be empty"),
    ...
])
def test_validation(attr, expected_message):
    attrs = dict(
        name='var_name',
        caption='Var Name',
        unit='m',
        visibility=Visibility.Output,
        location=Location.Center,
        multifield_scope=Scope.Global,
        checked_on_gui_default=False,
    )
    attrs.update(attr)
    with pytest.raises(TypeError, match=expected_message):
        Variable(**attrs)
```